doc/application/index.rst

```diff
@@ -251,7 +251,9 @@ should know about.
    3 ways (in order of precedence):
 
    * As a parameter to the ``west build`` or ``cmake`` invocation via the
-     ``-D`` command-line switch
+     ``-D`` command-line switch. If you have multiple overlay files, you should
+     use quotations. For west ``"file1.overlay file2.overlay"`` and for cmake
+     ``"file1.overlay;file2.overlay"``
```

Comment:
`west` and `cmake` uses same style.
So this is sufficient:
```suggestion
    use quotations, ``"file1.overlay;file2.overlay"``
```	


Just as a small side note: On windows, the quotation marks `""` are not required, as `;` is not a command separator on the Windows command prompt. (but Windows accepts quotation so using them is a safe bet for all users.)